user-guide/Troubleshooting/Procedures/STaaS_Troubleshooting.md

```diff
@@ -0,0 +1,265 @@
+
+- DataMiner version 10.4.0 or higher.
+
+- [CloudGateway DxM](#install-or-upgrade-the-cloudgateway-dxm) version 2.8.0 or higher, deployed on at least one DataMiner Agent.
+
+- A DataMiner System [connected to dataminer.services](#verify-your-dms-is-connected-to-dataminerservices).
+
+- A [working internet connection](#verify-your-dma-has-a-working-internet-connection).
+
+#### Install or upgrade the CloudGateway DxM
+
+To install the *CloudGateway* module:
+
+1. In the Admin app, check whether the correct organization is mentioned in the header bar.
+
+   > [!TIP]
+   > See also: [Accessing the Admin app](xref:Accessing_the_Admin_app)
+
+1. If a different organization should be selected, click the organization selector ![Organization selector](~/user-guide/images/Cloud_Admin_Selector_icon.png) in the top-right corner and select the organization in the list.
+
+1. In the pane on the left, under *DataMiner Systems*, select your DataMiner System and select the *DxMs* page.
+
+1. Locate the relevant node (i.e. the DMA).
+
+1. Next to the *CloudGateway* module, click *Deploy* to start the automatic installation process.
+
+If the *CloudGateway* module is installed on your DMA already, **verify that it is version 2.8.0 or higher**:
+
+- **Using the Admin app**:
+
+  1. On the *DxMs* page, locate the relevant node (i.e. the DMA).
+
+  1. Verify that the current version of the *CloudGateway* DxM is 2.8.0 or higher.
+
+  1. If necessary, click the *Upgrade* button to upgrade the module to a more recent version.
+
+- **Using SLLogCollector**:
+
+  1. Run the [SLLogCollector tool](xref:SLLogCollector).
+
+  1. Open the resulting package and navigate to *Logs* > *DxM* > *DataMiner CloudGateway* > *DataMiner CloudGateway.exe_version.txt*.
+
+  1. Confirm the CloudGateway DxM version under *Product Version*.
+
+- **Using the Skyline Admin app** (for Skyline Technical Support only):
+
+  1. In the [Skyline Admin app](https://skyline-admin.dataminer.services/organization), go to the *Organizations* tab and enter your organization in the search box to filter the results.
+
+  1. Click the eye icon next to your organization to access an overview of all DataMiner Systems created under it.
+
+  1. Locate your DMS and click the eye icon next to it.
+
+  1. Select the node (i.e. the DMA) on which the *CloudGateway* module is installed to expand the overview of deployed DxMs. This includes details such as DxM version, DxM data timestamp, and extra data.
+
+  1. Find the *DataMiner CloudGateway* DxM in the overview and verify that its version is 2.8.0 or higher.
+
+#### Verify your DMS is connected to dataminer.services
+
+> [!TIP]
+> See also: [Connecting your DataMiner System to dataminer.services](xref:Connecting_your_DataMiner_System_to_the_cloud)
+
+There are two ways to verify whether your DMS is connected to dataminer.services:
+
+- **Using DataMiner Cube**:
+
+  1. Go to the System Center > *Cloud* page, and select *Open dataminer.services*.
```

Comment:
This differs slightly from the steps in your original document. When I tested it, I couldn’t find any mention of "Session is active." Instead, I included an alternative way to verify the cloud connection on the same page.